tests/bluefinClient.test.ts

```diff
@@ -276,6 +262,180 @@ describe("BluefinClient", () => {
     });
   });
 
+  describe("Sub account Tests For One Click Trading", () => {
```

Comment:
Just realized these are all the same test cases from `"Sub account Tests"`. I don't see any major difference in them except here you use the `upsertSubAccount` call.

Given we have tests for the actual subaccount functionality, I think you only need to add tests for the new helper functions you created. So essentially just verifying that the `upsertSubAccount` function works as expected

Reply:
I added this flow to verify how the entire 1CT flow will work functionally so that after calling the upsertSubAccount everything should work like it is expected to. But I agree with your point of duplicacy of the same test cases. Let me make these changes, too.